src/quo/components/drawers/drawer_top/view.cljs

```diff
@@ -86,6 +86,18 @@
      :community-logo community-logo
      :size           24}]])
 
+(defn- account-context-tag-subtitle
+  [{:keys [account-name emoji customization-color]}]
+  [rn/view
+   {:accessibility-label :account-context-tag-wrapper
+    :style               {:flex-wrap :wrap}}
+   [context-tag/view
+    {:type                :account
+     :account-name        account-name
+     :emoji               emoji
+     :size                24
+     :customization-color customization-color}]])
```

Comment:
@J-Son89 
Is it okay to pass `customization-color` from top to bottom like this?

Reply:
Yeah it's never ideal to drill props but this is not the worst so I think it's okay to do like this :)